app/server/services/sets.ts

```
@@ -0,0 +1,43 @@
+import { supabase } from "@/lib/supabase";
+import { Set } from "@/types/DatabaseModels";
```

Biome (1.9.4)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)